src/actions/editorialWorkflow.js

```diff
@@ -1,4 +1,4 @@
-import uuid from 'uuid';
+import uuidv4 from 'uuid/v4';
```

Comment:
Nitpick: let's still refer to it as `uuid`, the implementation detail isn't relevant across the source.

Reply:
Changed.